test/end-to-end/company.js

```diff
@@ -100,6 +100,32 @@ describe("Company application endpoint", () => {
                 fs.unlinkSync(filename);
             });
 
+            test("should finish the application with single contact", async () => {
+                await test_agent
+                    .post("/company/application/finish")
+                    .attach("logo", "test/data/logo-niaefeup.png")
+                    .field("bio", "A very interesting and compelling bio")
+                    .field("contacts", "contact1")
+                    .expect(HTTPStatus.OK);
+
+                const test_company = [... await Company.find({}).exec()][0];
```

Comment:
No need for `.exec()`

```suggestion
                const test_company = [... await Company.find({})][0];
```